doc/release_notes/release_notes_2.0.rst

```diff
@@ -0,0 +1,285 @@
+
+Pre-launched Safety VM Support
+==============================
+
+ACRN supports a pre-launched partitioned safety VM, isolated from the
+Service VM and other post-launched VM by using partitioned HW resources.
+For example, in the hybrid mode, a real-time Zephyr RTOS VM can be
+“pre-launched” by the hypervisor even before the Service VM is launched,
+and with its own dedicated resources to achieve a high level of
+isolation. This is designed to meet the needs of a Functional Safety OS. 
+
+Post-launched VM support via OVMF
+=================================
+
+ACRN supports Open Virtual Machine Firmware (OVMF) as a virtual boot
+loader for the Service VM to launch post-launched VMs such as Windows,
+Linux, VxWorks, or Zephyr RTOS. Secure boot is also supported. 
+
+Post-launched Real-time VM Support
+==================================
+
+ACRN supports a post-launched RTVM, which also uses partitioned hardware
+resources to ensure good real-time performance, as required for
+industrial use cases.
+
+Real-Time VM Performance Optimizations
+======================================
+
+ACRN 2.0 improves (preempt-RT kernel-based) RTVM’s performance with
+these optimizations:
+
+* **Reduce the impact of VM-Exit overhead:**
+   Use Local APIC (LAPIC) passthrough, Virtio Polling Mode Drivers (PMD),
+   and NMI interrupt notification technologies. 
+
+* **Isolate the RTVM from the Service VM:**
+   Use Cache Isolation via Catch Allocation Technology (CAT) to ensure the
+   RTVM’s CPU using dedicated cache resource, avoiding interference from
+   other VMs  
+
+* **PCI Configuration space access emulation for passthrough devices in the hypervisor:**
+   The hypervisor provides the necessary emulation (such as config space)
+   of the passthrough PCI device during runtime for a DM-launched VM from
+   Service VM.  
+
+* **PCI bridge emulation inside hypervisor**
+
+* **ART (Always Running Timer Virtualization):**
+   Ensure time is synchronized between Ptdev and vART 
+
+CPU Sharing Support
+===================
+
+ACRN supports CPU Sharing to fully utilize the physical CPU resource
+across more virtual machines. ACRN enables a borrowed virtual time CPU
+scheduler in the hypervisor to make sure the physical CPU can be shared
+between VMs and support for yielding an idle vCPU when it’s running a
+‘HLT’ or ‘PAUSE’ instruction. 
+
+Many choices for User VM OS
+===========================
+
+ACRN now supports Windows* 10, Android*, Ubuntu*, Xenomai, VxWorks*,
+Real-Time Linux*, and Zephyr* RTOS.  ACRN’s Windows support now conforms
+to the Microsoft* Hypervisor Top-Level Functional Specification (TLFS).
+ACRN 2.0 also improves overall Windows as a Guest (WaaG) stability and
+performance. 
+
+Single Root I/O Virtualization (SR-IOV) Support
+===============================================
+
+SR-IOV allows a device to separate access to its resources among various
+PCIe hardware functions. For a network adapter, this enables network
```

Comment:
I'm not sure this is the right definition for SR-IOV. Isn't it more like "SR-IOV allows a single physical device to appear as multiple physical devices of the same type"?